PocketKit/Sources/PocketKit/NotificationRelay.swift

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Localization
+import Textile
+import Combine
+import SharedPocketKit
+
+/// Listens for general notifications and reroutes them to the appropriate NotificationCentre instance
+class NotificationRelay {
+    private let broadcastNotificationCentre: NotificationCenter
+    private var subscriptions: Set<AnyCancellable> = []
+
+    private var serverErrorActions: [Int: () -> Void] = [:]
+
+    init(_ broadcastNotificationCentre: NotificationCenter) {
+        self.broadcastNotificationCentre = broadcastNotificationCentre
+
+        registerSubscriptions()
+        registerActions()
```

Comment:
I think actions should come before subscriptions

Reply:
That's the tightest corner-case I ever did see.